tabbycat/participants/tables.py

```diff
@@ -56,6 +56,8 @@ def get_table(cls, view, participant):
                 queryset=DebateAdjudicator.objects.select_related('adjudicator__institution')),
             'debate__debateteam_set__team__speaker_set',
             'debate__round__motion_set',
+            Prefetch('debate__ballotsubmission_set',
+                queryset=BallotSubmission.objects.exclude(discarded=True)),
```

Comment:
I definitely like the efficiency thought here! But since it's used in a different file (utils/tables.py) and returns a different collection of objects to what `ballotsubmission_set` normally would, I'd prefer it if this particular call to `Prefetch()` used the `to_attr=` argument to put it in a different attribute name. As well as being explicit, this is a fail-fast measure: since `TabbycatTableBuilder.add_debate_ballot_link_column()` no longer filters out discarded ballots, if it's (now or in the future) accidentally called by a table that _hasn't_ made the effort to filter out discarded ballots ahead of time, I'd like it to fail straight away (rather than wait for some unsuspecting tournament to discover the bug).